org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/WProcessCtl.java

```diff
@@ -116,7 +116,8 @@ public static void process (int WindowNo, ProcessInfo pi, Trx trx, EventListener
 				para.addEventListener(DialogEvents.ON_WINDOW_CLOSE, new EventListener<Event>() {
 					@Override
 					public void onEvent(Event event) throws Exception {
-						parent.hideMask();
+						if(para.getDesktop() != null)
```

Comment:
I think it is better to use if (parent.getDesktop() != null) here

Reply:
But since parent is ISupportMask, i need to cast it to AbstractComponent. Should I do it that way?

Reply:
ic, in that case, it is better to use para then. actually, I don't think this change is needed at all if you have put in the infopanel fix.

Reply:
hmm .. since this is ON_WINDOW_CLOSE event, para.getDesktop() would always return null. why do need to add this check here ? if it is needed, please post the stack trace to the ticket.